src/cpp/rtps/writer/StatefulWriter.cpp

```diff
@@ -681,9 +730,9 @@ bool StatefulWriter::matched_reader_lookup(GUID_t& readerGuid,ReaderProxy** RP)
     return false;
 }
 
-bool StatefulWriter::is_acked_by_all(const CacheChange_t* change) const
+bool StatefulWriter::is_acked_by_all(const CacheChange_t* change)
```

Comment:
If `const` was removed because the mutex, make it mutable instead of removing the `const`.